src/engine/engine.cpp

```diff
@@ -401,7 +401,7 @@ const char* DivEngine::getSystemName(DivSystem sys) {
     case DIV_SYSTEM_PCE:
       return "PC Engine/TurboGrafx-16";
     case DIV_SYSTEM_NES:
-      return "NES";
+      return "NES/Famicom";
```

Comment:
This does add a mention of Famicom, but makes the name longer.

Reply:
will revert